content/en/docs/concepts/storage/volumes.md

```diff
@@ -541,29 +371,42 @@ The supported values for field `type` are:
 | Value | Behavior |
 |:------|:---------|
 | | Empty string (default) is for backward compatibility, which means that no checks will be performed before mounting the hostPath volume. |
-| `DirectoryOrCreate` | If nothing exists at the given path, an empty directory will be created there as needed with permission set to 0755, having the same group and ownership with Kubelet. |
+| `DirectoryOrCreate` | If nothing exists at the given path, an empty directory is created there as needed with permission set to 0755, having the same group and ownership with Kubelet. |
 | `Directory` | A directory must exist at the given path |
-| `FileOrCreate` | If nothing exists at the given path, an empty file will be created there as needed with permission set to 0644, having the same group and ownership with Kubelet. |
+| `FileOrCreate` | If nothing exists at the given path, an empty file is be created there as needed with permission set to 0644, having the same group and ownership with Kubelet. |
 | `File` | A file must exist at the given path |
 | `Socket` | A UNIX socket must exist at the given path |
 | `CharDevice` | A character device must exist at the given path |
 | `BlockDevice` | A block device must exist at the given path |
 
 Watch out when using this type of volume, because:
 
-* Pods with identical configuration (such as created from a PodTemplate) may
+* Pods with identical configuration (such as those created from a PodTemplate) may
   behave differently on different nodes due to different files on the nodes
 * The files or directories created on the underlying hosts are only writable by root. You
   either need to run your process as root in a
-  [privileged Container](/docs/tasks/configure-pod-container/security-context/) or modify the file
+  [privileged container](/docs/tasks/configure-pod-container/security-context/) or modify the file
   permissions on the host to be able to write to a `hostPath` volume
 
-#### hostPath configuration example
+#### Resources
 
-```yaml
-apiVersion: v1
-kind: Pod
-metadata:
+The storage media (such as Disk or SSD) of an `emptyDir` volume is determined by the
+medium of the filesystem holding the kubelet root dir (typically
+`/var/lib/kubelet`). There is no limit on how much space an `emptyDir` or
+`hostPath` volume can consume, and no isolation between containers or between
+pods.
+
+In the future, we expect that `emptyDir` and `hostPath` volumes will be able to
+request a certain amount of space using a [resource](/docs/concepts/configuration/manage-resources-containers/)
+specification, and to select the type of media to use, for clusters that have
+several media types.
```

Comment:
We usually [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future).

Reply:
I've revised this section with more updated information, based on https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#setting-requests-and-limits-for-local-ephemeral-storage